docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -4,18 +4,23 @@ skinparam arrowThickness 1.1
 skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
-AddressBook *-right-> "1" UniquePersonList
-AddressBook *-right-> "1" UniqueTagList
-UniqueTagList -[hidden]down- UniquePersonList
-UniqueTagList -[hidden]down- UniquePersonList
+Class I #FFFFFF
+Class "{abstract}\nPerson" as Person
 
-UniqueTagList -right-> "*" Tag
-UniquePersonList -right-> Person
+AddressBook *--> "1" UniquePersonList
+AddressBook *-left-> "1" UniqueLessonList
+UniquePersonList -right->"~* all" Person
```

Comment:
Is there aggregation here?

Reply:
Added aggregation for both UniquePersonList and UniqueLessonList. Good catch!